test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -850,6 +850,8 @@ BOOST_AUTO_TEST_CASE(cyclic_inheritance)
 		contract A is B { }
 		contract B is A { }
 	)";
+	// Error (TypeError): Definition of base has to precede definition of derived contract
+	// Warning: Use of var keyword is deprecated
```

Comment:
Why did you put this here?

Reply:
Not sure how that got in there, but I'll get rid of it.